README.md

```diff
@@ -7,7 +7,12 @@ To modify the website, please:
 
 * Create a fork and branch, and
 * add your proposed content.
-* Test your changes by [installing and running Jekyll locally](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/).
+* Test your changes by running
+```sh
+bundle install
```

Comment:
I have bundler installed via `dnf`. I guess we could enumerate the different ways to install bundler, but it seems like that is too much detail.